.gitignore

```diff
@@ -1,6 +1,6 @@
 # custom
 build/
-server/_secrets
+server/_secrets.ts
```

Comment:
I see you changed the name of the secrets file. I guess i will adjust on my end.